components/OfferSearch.tsx

```diff
@@ -91,10 +91,12 @@ export const OfferSearch = ({
 					return it.map((refinement) => ({
 						...refinement,
 						label:
-							listQuestion.find(
-								(it: any) =>
-									it.id === parseIdFromFacetName(refinement.attribute)
-							)?.[key] ?? refinement.label,
+							listQuestion.find((it: any) => {
+								if (it.type === "district") {
```

Comment:
o trochu niz se pouziva zase tohle: `["district"].includes(it.type)`

asi by to chtelo sjednotit a vyhodit ven, udelat pole neprekladanych klicu a mit jedno misto, kde se pro otazku rozhodne co prelozit a co ne, protoze driv nebo pozdeji pribude dalsi neprekladany klic :)